packages/expo-app-auth/src/AppAuth.ts

```diff
@@ -0,0 +1,135 @@
+import invariant from 'invariant';
+
+import { UnavailabilityError } from 'expo-errors';
+import ExpoAppAuth from './ExpoAppAuth';
+import {
+  OAuthServiceConfiguration,
+  OAuthDisplayParameter,
+  OAuthPromptParameter,
+  OAuthNonceParameter,
+  OAuthUILocalesParameter,
+  OAuthIDTokenHintParameter,
+  OAuthMaxAgeParameter,
+  OAuthLoginHintParameter,
+  OAuthACRValuesParameter,
+  OAuthParameters,
+  OAuthBaseProps,
+  OAuthProps,
+  OAuthRevokeOptions,
+  TokenResponse,
+} from './AppAuth.types';
+export {
+  OAuthServiceConfiguration,
+  OAuthDisplayParameter,
+  OAuthPromptParameter,
+  OAuthNonceParameter,
+  OAuthUILocalesParameter,
+  OAuthIDTokenHintParameter,
+  OAuthMaxAgeParameter,
+  OAuthLoginHintParameter,
+  OAuthACRValuesParameter,
+  OAuthParameters,
+  OAuthBaseProps,
+  OAuthProps,
+  OAuthRevokeOptions,
+  TokenResponse,
+};
+
+const isValidString = (s?: string): boolean => !!(s && typeof s === 'string');
```

Comment:
rename these functions to `assertValidString` etc... `isX` sounds like it returns true/false